static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -301,10 +336,18 @@ const Container = styled('div')`
   position: relative;
 
   & > div {
-    padding: 20px 0px;
+    padding: 10px 0px;
```

Comment:
should we use some of the space we have in our styles? https://github.com/getsentry/sentry/blob/master/static%2Fapp%2Fstyles%2Fspace.tsx